unit-tests/hex-content.js

```diff
@@ -0,0 +1,67 @@
+import test from 'tape'
+import React from 'react'
+import configure from 'enzyme-adapter-react-helper'
+import { shallow } from 'enzyme'
+import HexContent from '../app/components/hex-content'
+
+configure()
```

Comment:
Refactoring this to something like:

```javascript
import React from 'react'
import { init, shallow } from `./lib/enzymed`
import HexContent from '../app/compoents/hex-content'

const test = init() // runs the enzyme config and returns tape
```

would make it harder to make a mistake in the test setup

Reply:
hm, i was thinking of doing something like `import { shallow, render } from '../helpers/enzyme';` that is a helper file that does `configure()` &c and re-exports enzyme methods. But I thought it was not a big deal hmm

i'll do it in separate commit 👍